src/content/docs/pt-br/basics/astro-pages.mdx

```diff
@@ -97,3 +100,75 @@ Para uma página customizada de erro 404, você pode criar um arquivo `404.astro
+:::
+
+Parciais são componentes de página localizados em `src/pages/` que não tem como objetivo serem renderizados em páginas completas.
+
+Assim como componentes localizados fora dessa pasta, esses arquivos não incluem a declaração `<!DOCTYPE html>` automaticamente, nem qualquer outro conteúdo do `<head>`, como estilos com escopo ou scripts.
+
+Porém, por estarem localizados no diretório especial `src/pages/`, o código HTML gerado fica disponível em uma URL que corresponde ao caminho do arquivo. Isso permite que uma biblioteca de renderização(ex: htmx, Stimulus, jQuery) possa acessa-lo no cliente e carregar sessões do HTML dinamicamente em uma página, sem a necessecidade de um recarregamento ou navegação entre páginas.
+
+Parciais, quando combinados com uma biblioteca de renderização, proporcionam uma alternativa às [Ilhas Astro](/pt-br/concepts/islands/) e [tags `<script>`](/pt-br/guides/client-side-scripts/) na construção de conteúdo dinâmico em Astro.
+
+Arquivos de página que podem exportar um valor(ex: `.astro`, `.mdx`) podem ser marcadas como parciais.
+
+Configure um arquivo no diretório `src/pages/` para ser uma parcial adicionando a seguinte exportação:
+
+```astro title="src/pages/parcial.astro" ins={2}
+---
+export const partial = true;
+---
+
+<li>Eu sou uma parcial!</li>
+```
+
+O `export const partial` deve ser identificável estaticamente. Ele pode conter os seguintes valores.
+
+- O booleano __`true`__.
+- Uma variável de ambiente usando import.meta.env como `import.meta.env.USE_PARTIALS`.
+
+### Usando com uma biblioteca
+
+Parciais são usadas para atualizar, dinamicamente, sessões de uma página, usando bibliotecas como [htmx](https://htmx.org/).
+
+O exemplo a seguir mostra um atributo `hx-post` direcionado para uma URL parcial. O conteúdo da página parcial será utilizado para atualizar o elemento HTML escolhido na página.
+
+```astro title="src/pages/index.astro" 'hx-post="/partials/clicked/"'
+<html>
+  <head>
+    <title>Minha página</title>
+    <script src="https://unpkg.com/htmx.org@1.9.6"
+      integrity="sha384-FhXw7b6AlE/jyjlZH5iHa/tTe9EpJ1Y55RjcgPbjeWMskSxZt1v9qkxLJWNJaGni"
+      crossorigin="anonymous"></script>
+  </head>
+</html>
+<section>
+  <div id="parent-div">Mude aqui</div>
+
+  <button hx-post="/parciais/clicado/"
+    hx-trigger="click"
+    hx-target="#parent-div"
+    hx-swap="innerHTML"
+  >
+      Clique em mim!
+  </button>
+</section>
+```
+A parcial `.astro` deve existir no caminho correspondente e incluir uma exportação definindo a página como uma parcial.
+
+```astro title="src/pages/parciais/clicado.astro" {2}
+---
+export const partial = true;
+---
+<div>Eu fui clicado!</div>
+```
+
+Visite a [documentação do htmx](https://htmx.org/docs/) para mais detalhes na utilização do mesmo.
```

Comment:
Common mistake, "mesmo" is not a "pronome pessoal". [Here](https://rockcontent.com/br/talent-blog/pronome-mesmo/) is a good short read about it

```suggestion
Visite a [documentação do htmx](https://htmx.org/docs/) para mais detalhes na utilização do mesmo.
```